specification/maps/data-plane/Creator/preview/2022-09-01-preview/features.json

```diff
@@ -0,0 +1,1074 @@
+        "operationId": "Features_Delete",
+        "x-ms-examples": {
+          "DeleteFeature": {
+            "$ref": "./examples/features/Features_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "./dataset.json#/parameters/DatasetId"
+          },
+          {
+            "$ref": "#/parameters/CollectionId"
+          },
+          {
+            "$ref": "#/parameters/FeatureId"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Feature Deleted."
+          },
+          "default": {
+            "$ref": "../../../Common/preview/2022-09-01-preview/common.json#/responses/default"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "LandingPageResult": {
+      "type": "object",
+      "required": [
+        "links"
+      ],
+      "properties": {
+        "ontology": {
+          "$ref": "../../../Common/preview/2022-09-01-preview/common.json#/definitions/Ontology"
+        },
+        "links": {
+          "description": "Links to other Features endpoints.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FeaturesEndpointLink"
+          }
+        }
+      }
+    },
+    "ApiDefinitionResult": {
+      "type": "object",
+      "required": [
+        "openapi",
+        "info",
+        "paths",
+        "components",
+        "tags"
+      ],
+      "properties": {
+        "openapi": {
+          "description": "The OpenAPI version.",
+          "type": "string",
+          "example": "3.0.1"
+        },
+        "info": {
+          "description": "The OpenAPI info object.",
+          "type": "object",
+          "properties": {
+            "title": {
+              "description": "The title of the API.",
+              "type": "string",
+              "example": "Azure Maps Web Feature Service"
+            },
+            "version": {
+              "description": "The version of the API.",
+              "type": "string",
+              "example": "2022-09-01-preview"
+            },
+            "description": {
+              "description": "The description of the API.",
+              "type": "string",
+              "example": "Azure Maps Web Feature REST APIs"
+            }
+          }
+        },
+        "paths": {
+          "description": "The OpenAPI paths object.",
+          "type": "object"
+        },
+        "components": {
+          "description": "The OpenAPI components object.",
+          "type": "object",
+          "properties": {
+            "parameters": {
+              "description": "The OpenAPI parameters object.",
+              "type": "object"
```

Comment:
Should this have `"additionalProperties": true` to denote it's a map (JSON object) and accepts more information? This will also help our code generation for SDKs.

Reply:
Great suggestion, thank you! I've updated it.